openid-sharedsignals-framework-1_0.md

```diff
@@ -563,6 +568,28 @@ configuration information.
 
 Transmitters have metadata describing their configuration:
 
+spec_version
+
+> OPTIONAL. A version identifying the implementer's draft or final specification implemented by the Transmitter. This includes the numerical portion of the spec version as described in the document {{NAMINGCONVENTION}}. If absent, the Transmitter is assumed to conform to "1_0-ID1" version of the specification.
```

Comment:
maybe for clarity, add to the end "(this document)"

so: "If absent, the Transmitter is assumed to conform to "1_0-ID1" version of the specification (this document)."

Reply:
Updated.